packages/esm-service-queues-app/src/home.test.tsx

```diff
@@ -41,7 +41,7 @@ describe('Home Component', () => {
     window.location = { pathname: '/some-path/screen' };
 
     render(<Home />);
-    expect(screen.getByRole('progressbar')).toBeInTheDocument();
+    expect(screen.getByText(/patients currently in queue/i)).toBeInTheDocument();
```

Comment:
@donaldkibet I don't really know why this test broke. Could you confirm that I'm not breaking anything that shouldn't be broken?

Reply:
Ok, I'll take the thumbs up as meaning "this change seems fine."